configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+                                                                            radius:poi.radius
+                                                                        identifier:poi.identifier];
+               [ACPPlaces processRegionEvent:region forRegionEventType:ACPRegionEventTypeEntry];
+           }
+
+           // add the region to our updated list
+           [updatedRegionsUserIsCurrentlyWithin addObject:poi.identifier];
+       }
+   }
+
+   // update user defaults with the new list
+   [[NSUserDefaults standardUserDefaults] setObject:updatedRegionsUserIsCurrentlyWithin forKey:@"regionsUserIsAlreadyIn"];
+}
+```
+
+{% endtab %}
+
+{% tab title="iOS (Swift)" %}
+
+This code sample shows handling the result provided in the callback of `getNearbyPoints(_ ofInterest: CLLocation, limit: UInt, callback: (([ACPPlacesPoi]?) -> Void)?, errorCallback: ((ACPPlacesRequestError) -> Void)?)`, an `[ACPPlacesPoi]`:
+
+```swift
+func handleUpdatedPOIs(_ nearbyPois:[ACPPlacesPoi]) {
+    // get the list of regions we know the user is already within from user defaults
+    let regionsUserIsCurrentlyWithin : [String] = UserDefaults.standard.array(forKey: "regionsUserIsAlreadyIn") as! [String]
+
+    // loop through new nearbyPoi and post entry events for pois that aren't already in our list
+    // also creating the new list of known regions that the user is in
+    var updatedRegionsUserIsCurrentlyWithin: [String] = []
+    for poi in nearbyPois {
+        // check if the user is in this poi
+        if poi.userIsWithin {
+            // the user is in the poi, now we need to make sure we haven't already recorded the entry event
+            if !regionsUserIsCurrentlyWithin.contains(poi.identifier!) {
+                let region = CLCircularRegion.init(center: CLLocationCoordinate2D.init(latitude: poi.latitude, longitude: poi.longitude), radius: CLLocationDistance(poi.radius), identifier: poi.identifier!)
+                ACPPlaces.processRegionEvent(region, for: .entry)
+            }
+
+            // add the region to our updated list
+            updatedRegionsUserIsCurrentlyWithin.append(poi.identifier!)
+        }
+    }
+
+    // update user defaults with the new list
+    UserDefaults.standard.set(updatedRegionsUserIsCurrentlyWithin, forKey: "regionsUserIsAlreadyIn")
+}
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+## Putting the Pieces Together
+
+The following code samples should help put the picture together. They show retrieving the current location of the device, triggering the necessary events, and ensuring that you don't get multiple entries for the same location on a single visit.
+
+{% hint style="important" %}
+Remember that these snippets are meant to be examples. Determining how to implement the above functionality is up to the developer, and the decision should be made considering best practices as recommended by the target operating system.
```

Comment:
Please reword to:
"These snippets are **only** examples. Developers must determine how they want to implement the functionality, and the decision should consider the best practices as recommended by the target operating system.